### PR TITLE
Added categorized folder in data. Please review the PR.

### PR DESCRIPTION
**Pull request by amoghku02**

### CODE REVIEW

**Comment by amoghku02** [comment]
Issue#10: Added categorized folder in data. Please review the PR.

**Comment by lucasace** [comment]
Heyy, great work a few changes though
There are some players who have played for multiple countries, Ex: Boyd Rankin who played for both England and Ireland, you have only put him under Ireland, We would like you to segregate his England matches and Ireland matches and put each of them in the respective country. There are other players like that as well, you need to ensure it doesn't cause a problem

File: data/categorized/bowl/Ireland/Boyd Rankin  (Ireland  England).csv [hunk 1]
```diff
@@ -0,0 +1,74 @@
+,Match_id,Score,Overs,Maidens,Wicket,Economy
```
Comment (lucasace) [inline data/categorized/bowl/Ireland/Boyd Rankin  (Ireland  England).csv:1]: This file needs to be changed , you can find the corresponding match names in allmatches.csv .

**Review by lucasace — CHANGES_REQUESTED** [review]
Segregate Ireland and England matches , and create Boyd Rankin for both Ireland and England

File: data/categorized/bowl/United States West Indies/Xavier Marshall  (United States  West Indies).csv [hunk 2]
```diff
@@ -0,0 +1,3 @@
+,Match_id,Score,Overs,Maidens,Wicket,Economy
```
Comment (lucasace) [inline data/categorized/bowl/United States West Indies/Xavier Marshall  (United States  West Indies).csv:1]: Same problem, you can ignore USA as a country as of now, and simply change it to West Indies

**Review by lucasace — CHANGES_REQUESTED** [review]

File: data/categorized/bowl/Netherlands South Africa/Roelof van der Merwe  (Netherlands  South Africa).csv [hunk 3]
```diff
@@ -0,0 +1,16 @@
+,Match_id,Score,Overs,Maidens,Wicket,Economy
```
Comment (lucasace) [inline data/categorized/bowl/Netherlands South Africa/Roelof van der Merwe  (Netherlands  South Africa).csv:1]: i dont think we have records for netherlands for now, just put him under South Africa for now

**Review by lucasace — CHANGES_REQUESTED** [review]

File: data/categorized/wk/England/Eoin Morgan  (England  Ireland).csv [hunk 4]
```diff
@@ -0,0 +1,2 @@
+,MATCH_ID,SCORE
```
Comment (lucasace) [inline data/categorized/wk/England/Eoin Morgan  (England  Ireland).csv:1]: Same thing as the first review

**Review by lucasace — CHANGES_REQUESTED** [review]

File: data/categorized/wk/New Zealand/Luke Ronchi  (New Zealand  Australia).csv [hunk 5]
```diff
@@ -0,0 +1,68 @@
+,MATCH_ID,SCORE
```
Comment (lucasace) [inline data/categorized/wk/New Zealand/Luke Ronchi  (New Zealand  Australia).csv:1]: SImilar issue again

File: data/categorized/zip/England/Eoin Morgan  (England  Ireland).csv [hunk 6]
```diff
@@ -0,0 +1,326 @@
+matchid,date
```
Comment (lucasace) [inline data/categorized/zip/England/Eoin Morgan  (England  Ireland).csv:1]: same again

File: data/categorized/zip/Ireland/Boyd Rankin  (Ireland  England).csv [hunk 7]
```diff
@@ -0,0 +1,126 @@
+matchid,date
```
Comment (lucasace) [inline data/categorized/zip/Ireland/Boyd Rankin  (Ireland  England).csv:1]: Same again

**Review by lucasace — CHANGES_REQUESTED** [review]

**Comment by lucasace** [comment]
There are many more files like those, I think u got the general idea

**Comment by lucasace** [comment]
Once you are done , do tell as there are a lot of files to review

**Comment by lucasace** [comment]
Heyy, since there's no update on this for a long time, Im changing this PR to a draft. you can continue to work on it once your done, ill change it back

**Pull request closed without merging**